iceoryx_examples/callbacks/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.5)
+project(example_callbacks)
```

Comment:
Example needs to be added to https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/README.md

Reply:
Folder of example needs to be added to list in https://github.com/eclipse-iceoryx/iceoryx/blob/318c92314c79764fa334a563423a2be17d1b2c8d/tools/iceoryx_build_test.sh#L50